app/services/reports/users_report_service.rb

```diff
@@ -32,4 +35,9 @@ def org_name_user_count
       .order("organisations.name")
       .pluck("organisations.name", "COUNT(users.id)")
   end
+
+  def total_user_count
+    User.left_joins(:organisation)
+      .pick("COUNT(users.id)")
```

Comment:
Since we're not using the organisation in this query, I think this can be simplified by using [ActiveRecord's count method](https://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-count):

```suggestion
    User.count
```